setup.md

```diff
@@ -169,6 +169,25 @@ conda install -c conda-forge plotnine
 ~~~
 {: .language-bash}
 
+### _(Optional)_ Installing with environment file
```

Comment:
It might be better to say "Alternative" rather than "Optional" and mention "Required packages" somewhere to make it clear that we're talking about installing required packages, not Miniconda/Anaconda